healthcare/dataset_create.go

```diff
@@ -19,14 +19,13 @@ import (
 	"context"
 	"fmt"
 	"io"
+	"time"
 
 	healthcare "google.golang.org/api/healthcare/v1"
 )
 
 // createDataset creates a dataset.
-func createDataset(w io.Writer, projectID, location, datasetID string) error {
-	ctx := context.Background()
-
+func createDataset(ctx context.Context, w io.Writer, projectID, location, datasetID string) error {
 	healthcareService, err := healthcare.NewService(ctx)
```

Comment:
IMO it's better to create the context in the sample and set a timeout explicitly. This also is required by https://googlecloudplatform.github.io/samples-style-guide/#minimal-arguments Users will have to do this themselves in any case.

See example here: https://github.com/GoogleCloudPlatform/golang-samples/blob/HEAD/storage/buckets/create_bucket_class_location.go#L39

Reply:
thanks for pointing that out, I'll prefer following the style guide